### PR TITLE
Remove duplicate ops tests in test_quantized_op.py

### PR DESCRIPTION
**Pull request by kit1980**
The duplicates are after https://github.com/pytorch/pytorch/pull/94170

### CODE REVIEW

**Review by izaitsevfb — APPROVED** [review]

**Review by malfet — APPROVED** [review]

**Review by jerryzh168 — APPROVED** [review]
sorry I thought I have already approved this

File: test/quantization/core/test_quantized_op.py [hunk 1]
```diff
@@ -658,7 +656,6 @@ def test_qthreshold(self, X, threshold, value):
         ops_under_test = {
             'native': torch.threshold,
             'nn.functional': torch.nn.functional.threshold,
-            'nn.quantized.functional': torch.ao.nn.quantized.functional.threshold,
```
Comment (albanD) [inline test/quantization/core/test_quantized_op.py:661]: Both these names are valid for this function. So I don't think it is ok to remove this @kit1980

Comment (kit1980) [reply test/quantization/core/test_quantized_op.py:661]: @albanD But the key of this dict - name  - is used only for printing, it doesn't test anything in addition to `'ao.nn.quantized.functional': torch.ao.nn.quantized.functional.threshold`:

        for name, op in ops_under_test.items():
            qY = op(qX, threshold, value)
            self.assertEqual(qY, qY_hat, msg=f"{name} qthreshold failed")

**Pull request closed without merging**